.editorconfig

```diff
@@ -25,3 +25,11 @@ indent_size = unset
 
 [/assets/blacklists/GRCh37-blacklist.bed]
 trim_trailing_whitespace = unset
+
+# ignore Readme
+[README.md]
+indent_style = unset
+
+# ignore python
+[*.{py}]
```

Comment:
I think you need to add md here

Reply:
Comparing with other `.editorconfig` the only `md` is [here](https://github.com/nf-core/chipseq/blob/1b42a379c896c47d5557b376daa83fbd0c985161/.editorconfig#L11) and it is already there.